tests/integration/molecule/misc_api_authentication/converge.yml

```diff
@@ -145,6 +167,35 @@
         that:
           - result is success
 
+    - name: List assets using test user API key
+      asset_info:
+        auth:
+          api_key: "{{ api_key_test_user }}"
+
+    - name: Fail to create an asset using test user API key
+      bonsai_asset:
+        auth:
+          api_key: "{{ api_key_test_user }}"
+        name: sensu/monitoring-plugins
+        version: 2.2.0-1
+      ignore_errors: true
+      register: result
+
+    - assert:
+        that:
+          - result is failed
+
+    - name: Fail to list entities using test user API key
```

Comment:
These two tests are there because I wanted to cover two aspects of RBAC: the first one makes sure that the task fails if the user the forbidden verb on a resource that she has partial access to (user can list assets, but cannot create new ones), and the other tests the failure on a resource with no permissions whatsoever.

And this does not test our collection directly, I just left them there because I already wrote them when playing around with the RBAC. I will update the descriptions and leave them both in. They still can make sure we detect at least some of the API breakage.